vocabularies/earth-science-data-category.ttl

```diff
@@ -98,6 +105,67 @@ esdc:geology a skos:Concept ;
+esdc:history-of-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The historical development of geologic knowledge, including the history of observations of geologic features, geological exploration, drilling, mapping, etc. (Modified from: Neuendorf et al., 2011)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "History of Geology"@en .
+    skos:scopeNote "Also includes: The history of theories to explain the origin of geologic features, the history of the organisation and development of geologic institutions and societies, and the biographical study of geologists. Not to be confused with historical geology."@en ;
+
+esdc:historical-geology a skos:Concept ; 
+    skos:altLabel "paleogeology"@en , 
+        "palaeogeology"@en ;
+    skos:broader esdc:geology ; 
+    skos:definition "The branch of geology concerned with the reconstruction and comprehension of the geological history of the Earth. (Neuendorf et al., 2011)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Historical Geology"@en .
+    skos:scopeNote "Includes geologic processes on the surface of, and within, the Earth, involving investigations into stratigraphy, paleontology and the associated evolution of life forms throughout the interval of the geologic time scale, geochronology, plate tectonics, structural geology, paleoenvironments and changing paleoclimates. It is not to be confused with history of geology."@en ;
+
+esdc:areal-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "The comprehensive study of an area, including geological and geographical features. (The Free Dictionary by Farlex, 06.05.2020)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Areal Geology"@en .
+
+esdc:surface-geology a skos:Concept ; 
+    skos:altLabel "surficial geology"@en ; 
+    skos:broader esdc:geology ; 
+    skos:definition " The study of rock formations, soils, unconsolidated sediments, geological structures and other features, as observed and recorded or mapped at or near the Earth's surface. (Modified from Neuendorf et al., 2011; www.mindat.org, 06.05.2020)."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Surface Geology"@en .
+
+esdc:field-geology a skos:Concept ; 
+    skos:broader esdc:geology ; 
+    skos:definition "Geology variously involving direct observation, identification, description and measurement of outcropping rocks, minerals and landforms, and the preparation and interpretation of geological cross-sections and maps. Includes assessment of stratigraphic, structural/tectonic and sedimentological features."@en ; 
+    # Can anyone provide a better definition?#
+    skos:inScheme <http://linked.data.gov.au/def/earth-science-data-category> ;
+    skos:prefLabel "Field Geology"@en .
```

Comment:
I'm unconvinced this is real. Field geology is more a method of research than a subject of research. It is one way to study Surface Geology, or Sedimentology, or Structural Geology etc.

Reply:
I was thinking more about what info was collected during field geology work, but I agree that we do have surface geology, and this will suffice.